src/cli/cli.ts

```diff
@@ -0,0 +1,47 @@
+const fs = require('fs');
+const path = require('path');
+import { Command } from 'commander';
+import {Board} from "../board/board"
+import chalk from "chalk";
+
+let program : Command;
+const CANNOT_READ_FILE_ERROR = "Can't read file; please check your -f argument point to a valid file."
+
+export const initCLI = () => {
+    program = new Command();
+    program
+    .option('-f, --filepath <string>', "A path to load a kamon game from a file")
+    .parse(process.argv);
+}
+
+export const loadGameConfigFromFile = ():Board => {
+    const filePath = loadFileFromParameters();
+    return loadFileFromPath(filePath);
+}
```

Comment:
I disagree with the way you organized the files. To me, `cli.ts` should only have the responsibility of initializing the CLI and parsing the arguments (i.e. get the file path as a string), but parsing the content of the file into a Board object is another feature that should be in its own file.

Reply:
Done. Only kept pure CLI parameters fetching into cli.ts and made a new file.